go/oasis-test-runner/oasis/args.go

```diff
@@ -32,6 +39,17 @@ import (
 	workerStorage "github.com/oasislabs/oasis-core/go/worker/storage"
 )
 
+const (
+	MetricsJobName = "oasis-test-runner"
```

Comment:
Where is this used?

Reply:
When pushing `oasis_up` metric.

Reply:
In that case, it should be moved there instead of being defined here?

Reply:
I renamed it to `MetricsJobTestRunner` and moved it to `metrics.go`. We might have more job names in the future.